Qt4_LimaVideo.py

```diff
@@ -1,343 +1,188 @@
-        """
-        image = self.video.getLastImage()
-        if image.frameNumber() > -1:
-            raw_buffer = image.buffer()
-            if self.cam_type == "basler":
-                raw_buffer = self.decoder(raw_buffer)
-                qimage = QtGui.QImage(raw_buffer, image.width(), image.height(),
-                                      image.width() * 3,
-                                      QtGui.QImage.Format_RGB888)
-            else:
-                qimage = QtGui.QImage(raw_buffer, image.width(), image.height(),
-                                      QtGui.QImage.Format_RGB888)
-            if self.cam_mirror is not None:
-                qimage = qimage.mirrored(self.cam_mirror[0], self.cam_mirror[1])     
-            qpixmap = QtGui.QPixmap(qimage)
-            self.emit("imageReceived", qpixmap)
-            return qimage
-
-    def y8_2_rgb(self, raw_buffer):
-        image = np.fromstring(raw_buffer, dtype=np.uint8)
-        image.resize(self.image_dimensions[1], self.image_dimensions[0], 1)
-        return cv2.cvtColor(image, cv2.COLOR_GRAY2RGB)
-
-    def yuv_2_rgb(self, raw_buffer):
-        image = np.fromstring(raw_buffer, dtype=np.uint8)
-        image.resize(self.image_dimensions[1], self.image_dimensions[0], 2)
-        return cv2.cvtColor(image, cv2.COLOR_YUV2RGB_UYVY)
-
-    def save_snapshot(self, filename, image_type='PNG'):
-        qimage = self.get_new_image() 
-        qimage.save(filename, image_type) 
-
-    def get_snapshot(self, bw=None, return_as_array=True):
-        qimage = self.get_new_image()
-        if return_as_array:
-            qimage = qimage.convertToFormat(4)
-            ptr = qimage.bits()
-            ptr.setsize(qimage.byteCount())
-            image_array = np.array(ptr).reshape(qimage.height(),
-                                                qimage.width(), 4)
-            if bw:
-                return np.dot(image_array[...,:3],[0.299, 0.587, 0.144])
-            else:
-                return image_array 
-            
-        else:
-            if bw:
-                return qimage.convertToFormat(QtGui.QImage.Format_Mono)
-            else:
-                return qimage
-
-    def get_contrast(self):
-        return
-
-    def set_contrast(self, contrast_value):
-        return
-
-    def get_brightness(self):
-        return
-
-    def set_brightness(self, brightness_value):
-        return
-
-    def get_gain(self):
-        if self.cam_type == "basler":
-            value = self.video.getGain()
-            return value
-
-    def set_gain(self, gain_value):
-        if self.cam_type == "basler":
-            self.video.setGain(gain_value)
-
-    def get_gamma(self):
-        return
-
-    def set_gamma(self, gamma_value):
-        return
-
-    def get_exposure_time(self):
-        if self.cam_type == "basler":
-            value = self.video.getExposure()
-            return value
-
-    def set_exposure_time(self, exposure_time_value):
-        return
+#

```

Comment:
This looks like an old version. Could you please update it to the one in master?